### ISSUE TITLE
Compiling RHVoice on OSX

### ISSUE THREAD

**Issue by yauhen-info** [nlprocby/RHVoice#3]
Quick draft:
http://www.ficksworkshop.com/blog/14-coding/65-installing-gcc-on-mac
Try to do

```
brew tap homebrew/versions
```

first, and then

```
brew install gcc49
```

https://github.com/Automattic/node-canvas/wiki/Installation---OSX
https://github.com/rviney/scons-mac-installer   

```
brew install scons
```

**Comment by yauhen-info** [comment]
Result is proved also by CI #1, #4 ; 

```
In file included from src/lib/lib.cpp:23:
In file included from src/include/core/document.hpp:31:
In file included from src/include/core/ssml.hpp:22:
src/include/core/xml.hpp:361:15: error: reference to non-static member function must be called
      xml_doc.parse<rapidxml::parse_non_destructive>(&xml_text[0]);
      ~~~~~~~~^~~~~
src/include/core/document.hpp:577:12: note: in instantiation of function template specialization 'RHVoice::xml::parser<char>::parse<const char *>' requested here
    parser.parse(text_start,text_end,*doc_ptr);
           ^
src/lib/lib.cpp:223:25: note: in instantiation of function template specialization 'RHVoice::document::create_from_ssml<const char *>' requested here
      doc_ptr=document::create_from_ssml(engine_ptr,text,text+length,profile);
                        ^
src/lib/lib.cpp:119:17: note: in instantiation of function template specialization 'RHVoice_message_struct::RHVoice_message_struct<char>' requested here
    return (new RHVoice_message_struct(engine_ptr,callbacks,text,length,message_type,synth_params,user_data));
                ^
src/lib/lib.cpp:329:39: note: in instantiation of function template specialization 'RHVoice_tts_engine_struct::new_message<char>' requested here
      return (tts_engine?(tts_engine->new_message(text,length,message_type,synth_params,user_data)):0);
                                      ^
src/third-party/rapidxml/rapidxml/rapidxml.hpp:1381:14: note: possible target for call
        void parse(Ch *text)
             ^
In file included from src/lib/lib.cpp:23:
In file included from src/include/core/document.hpp:31:
In file included from src/include/core/ssml.hpp:22:
src/include/core/xml.hpp:361:15: error: reference to non-static member function must be called
      xml_doc.parse<rapidxml::parse_non_destructive>(&xml_text[0]);
      ~~~~~~~~^~~~~
src/include/core/document.hpp:577:12: note: in instantiation of function template specialization 'RHVoice::xml::parser<wchar_t>::parse<const wchar_t *>' requested here
    parser.parse(text_start,text_end,*doc_ptr);
           ^
src/lib/lib.cpp:223:25: note: in instantiation of function template specialization 'RHVoice::document::create_from_ssml<const wchar_t *>' requested here
      doc_ptr=document::create_from_ssml(engine_ptr,text,text+length,profile);
                        ^
src/lib/lib.cpp:119:17: note: in instantiation of function template specialization 'RHVoice_message_struct::RHVoice_message_struct<wchar_t>' requested here
    return (new RHVoice_message_struct(engine_ptr,callbacks,text,length,message_type,synth_params,user_data));
                ^
src/lib/lib.cpp:343:39: note: in instantiation of function template specialization 'RHVoice_tts_engine_struct::new_message<wchar_t>' requested here
      return (tts_engine?(tts_engine->new_message(text,length,message_type,synth_params,user_data)):0);
                                      ^
src/third-party/rapidxml/rapidxml/rapidxml.hpp:1381:14: note: possible target for call
        void parse(Ch *text)
             ^
2 errors generated.
scons: *** [build/darwin/lib/lib.os] Error 1
scons: building terminated because of errors.
The command "scons" exited with 2.
```

**Comment by abitrolly** [comment]
Fixed by 523262d. Thanks to @agutikov. =)
